app/client/mindjogg/screens/emergency_support/EmergencySupportMainScreen.js

```diff
@@ -1,99 +1,119 @@
-import React from "react";
-import { 
-  Text, 
-  View, 
-  StyleSheet, 
-  TouchableOpacity, 
+import React, { useEffect, useState } from "react";
+import {
+  Text,
+  View,
+  StyleSheet,
+  TouchableOpacity,
   Dimensions,
   ScrollView,
   Alert,
- } from "react-native";
+} from "react-native";
+import axios from "axios";
 import { globalStyles } from "../../styles/global";
+import StdCard from "../../components/StdCard/StdCard";
 
+var count = 0;
 
-const askHelp = async () => {
+const askHelp = () => {
   Alert.alert("Calling 911 ...");
-  
 };
 
+const EmergencySupportMainScreen = () =>
+  // {
+  //   /*navigation*/
+  // }
+  {
+    const [emergList, setEmergList] = useState([]);
 
-const EmergencySupportMainScreen = ({navigation}) => {
-  return (
-    <View style={styles.container}>
-      
-      <View style={styles.sosButton}>
-          <Text style={styles.headerTitleText}>
-            Immediate Help Needed?
-          </Text>
-            <TouchableOpacity
-              style={styles.circle}
-              onPress={() => {askHelp()}}
-            >
-              <Text style={styles.sosButtonText}> Call 911 </Text>
-            </TouchableOpacity>
-      </View>
+    /**
+     * Gets a List of Emergency Services
+     */
+    const retrieveServices = async () => {
+      try {
+        //Change the IP address to your Local Address
+        var service = await axios.get(
+          "http://192.168.2.35:8080/emergency/list",
```

Comment:
Would you mind putting your IP in a constant? Something like this:
`const BACKEND = "192.168.2.35:8080;
get(BACKEND + "/emergency/list`
Just to make it easier to change in the future

Other than that, this looks good to merge

Reply:
Will attend to now